tap_trello/streams.py

```diff
@@ -220,30 +220,38 @@ def get_records(self, format_values, additional_params=None):
             )
 
         for rec in records:
-            yield self.modify_record(rec, parent_id_list = format_values, custom_fields_map = custom_fields_map)
+            yield self.modify_record(rec, parent_id_list = format_values, custom_fields_map = custom_fields_map, dropdown_fields_map = dropdown_fields_map)
 
 
     def sync(self):
         for rec in self.get_records(self.get_format_values()):
             yield rec
 
 class AddCustomFields(Mixin):
-    def build_custom_fields_map(self, **kwargs):
+    def build_custom_fields_maps(self, **kwargs):
         custom_fields_map = {}
+        dropdown_options_map = {}
         board_id_list = kwargs['parent_id_list']
         # The custom fields are defined on the board level, so this function is called on a per-board basis
         # Therefore, we assert that only one board is being passed in
         assert len(board_id_list) == 1
         custom_fields = self.client.get('/boards/{}/customFields'.format(board_id_list[0])) # pylint: disable=no-member
         for custom_field in custom_fields:
             custom_fields_map[custom_field['id']] = custom_field['name']
-        return custom_fields_map
+            if custom_field['type'] == 'list':
+                for dropdown_option in custom_field['options']:
+                    dropdown_options_map[dropdown_option['id']] = dropdown_option['value']['text']
```

Comment:
We're assuming that these don't collide across custom fields, which I think is a good assumption, but is there a way to nest it if we need to?

Reply:
Talked about it, concatenation of fieldId and optionId seems to be the most straightforward way to go.